sql/src/main/java/io/crate/operation/collect/collectors/RemoteCollector.java

```diff
@@ -81,11 +83,12 @@ public RemoteCollector(UUID jobId,
         this.localNode = localNode;
         this.remoteNode = remoteNode;
 
+        this.scrollRequired = rowReceiver.requirements().contains(Requirement.REPEAT);
```

Comment:
btw: this is one of many cases where it is still interesting to have the batchcosumer know if it requires scroll